life_expectancy/tests/test_main.py

```diff
@@ -0,0 +1,21 @@
+"""Tests for the cleaning module"""
+from unittest.mock import patch
+import pandas as pd
+
+from life_expectancy.main import main
+from . import FIXTURES_DIR
+
+def test_main(pt_life_expectancy_expected_test):
+    """Run the `clean_data` function and compare the output to the expected output"""
+
+    main(
+        country='PT',
+        input_file_path=FIXTURES_DIR / "eu_life_expectancy_raw_test.tsv",
```

Comment:
Calling the main function like this, would still use the save_data and changing the file contents right? Maybe changing the functionality of save_data with a monkeypatch to print :). Then changing the main function to return the cleaned data and compare that instead of reading the csv.